frontend/javascripts/libs/UpdatableTexture.ts

```diff
@@ -9,25 +10,39 @@ const lazyGetCanvas = _.memoize(() => {
   return canvas;
 });
 
-const getImageData = _.memoize((width: number, height: number): ImageData => {
-  const canvas = lazyGetCanvas();
-  const ctx = canvas.getContext("2d");
-  if (ctx == null) {
-    throw new Error("Could not get context for texture.");
-  }
-  const imageData = ctx.createImageData(width, height);
+const getImageData = _.memoize(
+  (
+    width: number,
+    height: number,
+    isInt: boolean,
+  ): { width: number; height: number; data: TypedArray } => {
+    const canvas = lazyGetCanvas();
+    const ctx = canvas.getContext("2d");
+    if (ctx == null) {
+      throw new Error("Could not get context for texture.");
+    }
 
-  // Explicitly "release" canvas. Necessary for iOS.
-  // See https://pqina.nl/blog/total-canvas-memory-use-exceeds-the-maximum-limit/
-  canvas.width = 1;
-  canvas.height = 1;
-  ctx.clearRect(0, 0, 1, 1);
+    if (isInt) {
+      return { width, height, data: new Uint32Array(4 * width * height) };
+    }
```

Comment:
this is a bit annoying. until now, we always used "normal" three js textures (inherited via UpdatableTexture) which requires an ImageData object. for integer textures this is not possible which is why we have to set `isDataTexture = true` and use this dummy image object for that case. we could always do this (independent of int textures), but then we'd need to have the distinction what kind of typed array we need to instantiate for which texture (combined with channel count).

A way easier way can be found here, though: https://github.com/scalableminds/webknossos/compare/segment-color-cuckoo...empty-texture-init (dev machine: https://emptytextureinit.webknossos.xyz/).
It always uses data textures and just returns `null` for the initial data value. This seems to work great on my machine. **However**, an irritating webgl warning is emitted on the console when doing this. That's why I'm hesitant to make this the go-to way.
@daniel-wer Do you have an opinion on this or an idea how to deal with this scenario better?